pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1215,8 +1217,7 @@ class Build {
                             if (buildConfig.DOCKER_FILE) {
                                 try {
                                     context.timeout(time: buildTimeouts.DOCKER_CHECKOUT_TIMEOUT, unit: "HOURS") {
-                                        def repoHandler = new RepoHandler(context, USER_REMOTE_CONFIGS)
-                                        repoHandler.checkoutAdopt()
+                                        context.checkout context.scm
```

Comment:
this will checkout ci-jenkins-pipelines rather than openjdk-build now?

Reply:
I suspect this needs to now be:
if (useAdoptShellScripts) {
    repoHandler.checkoutAdopt()
} else {
    repoHandler.checkoutUser()
}